### PR TITLE
Added my code

### PR DESCRIPTION
**Pull request by MiriamOkpalaeke**
my main work is on the branch Miriam

### CODE REVIEW

File: film app/src/Styling/Main.css [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  background-image: url('../Assets/bg-card-4.c21c4d4b.png');
+}
+
+.movie-card:nth-child(5) {
+  background-image: url('../Assets/bg-card-5.67d061d6.png');
+}
+
+.movie-card:nth-child(6) {
+  background-image: url('../Assets/bg-card-6.b4fea25d.png');
+}
```
Comment (tundeojediran) [inline film app/src/Styling/Main.css:72]: Nice work with the background image implementation

File: film app/src/Components/MovieList.jsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      })
+      .catch((error) => {
+        console.log(error);  
+        setError(null)
+        setData(null)
+      })
+      .finally(() => {
+        setLoading(false);
+      });
+  },[]);
```
Comment (tundeojediran) [inline film app/src/Components/MovieList.jsx:40]: Great job handling the data fetching side-effect in the useEffect

**Review by tundeojediran — APPROVED** [review]
Overall, you did an amazing job. Kudos!!!

**Comment by MiriamOkpalaeke** [comment]
merging

**Pull request merged**